src/server/aws-ecr-policy.ts

```diff
@@ -0,0 +1,28 @@
+import { type AwsIAMPolicy, AwsEcrActions as Action } from 'aws-iam-policy-types'
+import { ENCLAVE_AWS_ACCOUNT_NUMBERS } from './config'
+
+export const EcrPolicy: AwsIAMPolicy = {
+    Version: '2012-10-17',
```

Comment:
I'm sure this has no side effects but my curiosity is wondering about the significance of this date 😆

Reply:
yah, it's AWS specific and refers to the schema for the IaM statement.  Your comment prompted me to look it up and it appears there's only two valid values: '2012-10-17' | '2008-10-17'